R/centrality.R

```diff
@@ -1132,28 +1102,43 @@ diversity <- diversity_impl
 #' hub_score(g2)$vector
 #' authority_score(g2)$vector
 #' @family centrality
-hub_score <- function(graph, scale=TRUE, weights=NULL, options=arpack_defaults()) {
+hits_scores <- function(graph, scale=TRUE, weights=NULL, options=arpack_defaults()) {
```

Comment:
should this get the ... argument then checked with `rlang::check_dots_empty()`?
```suggestion
hits_scores <- function(graph, ..., scale=TRUE, weights=NULL, options=arpack_defaults()) {
```